asab/api/service.py

```diff
@@ -60,14 +59,10 @@ def _build_zookeeper_adv_data(self):
 		return adv_data
 
 
-	def _initialize_web(self, listen):
+	def _initialize_web(self):
 		websvc = self.App.get_service("asab.WebService")
 
-		# Create a dedicated web container
-		container = asab.web.WebContainer(
-			websvc, "asab:web",
-			config={"listen": listen}
-		)
+		container = websvc.Containers.get(websvc.WebConfigName)
```

Comment:
`container.WebApp.Container` (no lookup I guess)

Reply:
aha, I guess you meant `websvc.WebApp.Container`. fixed